### DISCUSSION TITLE
confusing with answer of "Can I use it with React Native?" FAQ from Documentation

### DISCUSSION THREAD

**Discussion by dzakki** [mswjs/msw#984] [Q&A] [answered]
[documentation](https://mswjs.io/docs/faq#can-i-use-it-with-react-native) told me if msw only support for testing in react native, but when I see the [code](https://github.com/mswjs/msw/blob/master/src/native/index.ts) msw supported for native and not only for testing. so, what's correct? 

I found an example for msw in react-native. 
https://github.com/wwdrew/examples/tree/react-native/examples/react-native

**Comment by kettanaito** [comment, marked as answer]
Hey, @dzakki. 
You can use MSW for both testing and development in React Native. I think we should adjust the wording in the referenced documentation passage. Would you like to submit a pull request?

**Reply by dzakki** [reply]
Hey @kettanaito ,
Thanks a lot for the answer!
i will be glad to submit a pull request for adjusting the wording in the referenced documentation passage. :slightly_smiling_face:
